lerna-package-lock.json

```diff
@@ -37727,7 +37722,7 @@
 		"is-extglob": {
 			"version": "2.1.1",
 			"resolved": "https://registry.npmjs.org/is-extglob/-/is-extglob-2.1.1.tgz",
-			"integrity": "sha512-SbKbANkN603Vi4jEZv49LeVJMn4yGwsbzZworEoyEiutsN3nJYdbO36zfhGJ6QEDpOZIFkDtnq5JRxmvl3jsoQ=="
+			"integrity": "sha1-qIwCU1eR8C7TfHahueqXc8gz+MI="
```

Comment:
grr, sha512->sha1 integrity changes. let me cache clear and reinstall. changes here should only be to pako.

Reply:
fixed